labgrid/driver/power/phoenix_fl_switch.py

```diff
@@ -0,0 +1,74 @@
+'''
+This Driver was tested on a FL SWITCH 2303-8SP1 with FW-version 3.27.01 BETA
+file   phoenix_fl_switch.py
+author Raffael Krakau
+date   2023-08-24
+
+Copyright 2023 JUMO GmbH & Co. KG
+'''
+import pexpect
+
+PORT = 23
+
+
+def __login_telnet(tn):
+    """
+    Login user with set credentials
+
+    @param tn : pyexpect-telnet-object
+    """
+    username = "admin"
+    password = "private"
```

Comment:
Instead of hardcoding the credentials, you could use a URL like `telnet://admin:private@1.2.3.4/` as the host. You'd need to add `telnet` as a scheme in https://github.com/labgrid-project/labgrid/blob/master/labgrid/driver/powerdriver.py#L194

Reply:
This would indeed be nice.

I used _netio_kshell.py_ as inspiration for the _phoenix_fl_switch.py_ .
Implementing a new scheme would be some effort.

If you want me to put this suggestion on our todo this will be noted ?

Reply:
Commits have been squashed as requested.